web/src/components/ExportEventButton.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { ReactElement, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import { EventModel } from 'shared/api'
+
+import buildConfig from '../constants/buildConfig'
+import dimensions from '../constants/dimensions'
+import useWindowDimensions from '../hooks/useWindowDimensions'
+import RadioGroup from './base/RadioGroup'
+import TextButton from './base/TextButton'
+
+const ButtonContainer = styled.div`
+  display: flex;
+  gap: 16px;
+
+  @media ${dimensions.smallViewport} {
+    flex-direction: column;
+  }
+`
+
+const CancelButton = styled(TextButton)<{ fullWidth: boolean }>`
+  ${props => props.fullWidth && 'width: 100%;'}
+  background-color: ${props => props.theme.colors.textDecorationColor};
+  margin: 0;
+`
+
+const StyledButton = styled(TextButton)<{ fullWidth: boolean }>`
+  ${props => props.fullWidth && 'width: 100%;'}
+  margin: 0;
+`
+
+type ExportEventButtonProps = {
+  event: EventModel
+}
+
+const ExportEventButton = ({ event }: ExportEventButtonProps): ReactElement => {
+  const [isExporting, setIsExporting] = useState<boolean>(false)
+  const [exportRecurring, setExportRecurring] = useState<boolean>(false)
+  const { t } = useTranslation('events')
+  const { viewportSmall } = useWindowDimensions()
+
+  const isRecurring = event.date.recurrenceRule && event.date.recurrenceRule.count() > 1
```

Comment:
I think it would perhaps be better to check using our own method. Here we might have edge cases with different timezones, durations and so on again.
```suggestion
  const isRecurring = date.hasMoreRecurrencesThan(1)
```